Sources/HAP/Base/Generated.swift

```diff
@@ -254,6 +350,215 @@ public extension CharacteristicType {
 	static let wiFiConfigurationControl = CharacteristicType(0x022D)
 }
 
+extension CharacteristicType: CustomStringConvertible {
```

Comment:
Same comment as `ServiceType.description`.

Reply:
OK Same reply

Reply:
Same remark regarding the hex conversion.